src/TestSuite/TestCase.php

```diff
@@ -488,6 +494,10 @@ protected function _assertAttributes($assertions, $string)
                 }
             }
             if ($matches === false) {
+                if ($fullDebug === true) {
+                    debug($string);
+                    debug($regex);
+                }
```

Comment:
If asserts didn't stop the tests none of this would've been a problem :) I think this might be leftover from the simple test days hah